packages/server/lib/clients/auth.client.ts

```diff
@@ -32,7 +33,7 @@ export function setupAuth(app: express.Router) {
             store: sessionStore,
             name: 'nango_session',
             unset: 'destroy',
-            cookie: { maxAge: 30 * 24 * 60 * 60 * 1000, secure: false },
+            cookie: { maxAge: 7 * 24 * 60 * 60 * 1000, secure: false },
```

Comment:
at some point we are gonna need to review our cookie policy. In the meantime 7 days is probably a good compromise

Reply:
yes, long sessions are a liability